activation/activation.go

```diff
@@ -228,6 +241,56 @@ func (b *Builder) buildNipstChallenge(epoch types.EpochId) error {
 	return nil
 }
 
+func (b *Builder) StartPost(rewardAddress address.Address, dataDir string, space uint64) error {
+	b.log.Info("Starting post, reward address: %x", rewardAddress)
+	b.SetCoinbaseAccount(rewardAddress)
+	b.postInitLock.Lock()
+	if b.initStatus == Done {
+		return nil
+	}
+	if b.initStatus != Idle {
+		b.postInitLock.Unlock()
+		return fmt.Errorf("attempted to start post when post already initiated")
+	}
+	b.initStatus = InProgress
+	errChan := make(chan error, 1)
+	go func() {
+		_, err := b.nipstBuilder.InitializePost(dataDir, space) // TODO: add proof to first ATX
+		b.postInitLock.Lock()
+		if err != nil {
+			b.initStatus = Idle
```

Comment:
You're not inspecting "already initialized" to set a `Done` status. 
If you don't want to do it now, leave a TODO.

Reply:
there is no "already initialized error, we use the `initStatus` flag to know this

Reply:
I was referring to an error which comes from `go-spacemesh/post` initializer. And my mistake - it will return "already completed" and not "already initialized" (that was the old version).

Reply:
then I think we should not return an error in such a case, and simply return the proof